src/components/MenuButton/MenuButton.css

```diff
@@ -0,0 +1,80 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+@import "src/css/variables/colors.css";
+
+.y-menu-button {
+  height: 20px;
+  width: 20px;
+  border: 0;
+  padding: 0;
+  opacity: 0.7;
+  color: $textColor__secondary;
+}
+.y-menu-button:hover,
+.y-menu-button:active,
+.y-menu-button:focus {
+  color: $linkColor;
+  opacity: 1;
+}
+
+.y-menu-button--callout {
```

Comment:
I think this is the opposite of BEM. If we were following BEM, this would be `y-menu-button__callout`.

Reply:
http://getbem.com/naming/